src/app/modules/form/admin-form/admin-form.service.ts

```diff
@@ -1044,3 +1044,28 @@ export const updateStartPage = (
     return okAsync(updatedForm.startPage)
   })
 }
+
+/**
+ * Disables sms verifications for all forms belonging to the specified user
+ * @param userId the id of the user whose sms verifications should be disabled
+ * @returns ok(true) when the forms have been successfully disabled
+ * @returns err(PossibleDatabaseError) when an error occurred while attempting to disable sms verifications
+ */
+export const disableSmsVerificationsForUser = (
+  userId: string,
+): ResultAsync<true, PossibleDatabaseError> =>
+  ResultAsync.fromPromise(
+    FormModel.disableSmsVerificationsForUser(userId).then(() => true),
```

Comment:
should the `.then` part be chained from ResultAsync instead?

```ts
ResultAsync.fromPromise(
  FormModel.disableSmsverificationsForUser(userId),
  (error) => { ... }
).map(() => true)
```

Reply:
thanks for suggestion, will change!